src/geophires_x/CylindricalReservoir.py

```diff
@@ -80,6 +83,7 @@ def __init__(self, model: Model):
         )
         self.RadiusOfEffect = self.ParameterDict[self.RadiusOfEffect.Name] = floatParameter(
             "Cylindrical Reservoir Radius of Effect",
+            value=30.0,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)